_data/hosting.yml

```diff
@@ -451,6 +451,13 @@ websites:
       - totp
     doc: https://support.stackpath.com/hc/en-us/articles/360022992631
 
+  - name: switchplus
```

Comment:
```suggestion
  - name: Swizzonic (formerly switchplus)
```
They have since changed names.

Reply:
Do we really want include the former name?

Reply:
I included their old name for the edge case that people may search the site use that name.

Reply:
The URL is included in the search terms, so using _Swizzonic_ as the name will be sufficient.